client/components/Navigation/navigation.jsx

```diff
@@ -1,13 +1,33 @@
 import { NavLink } from 'react-router-dom';
 import styles from './navigation.module.css';
-import linkStyles from '../link.module.css';
+// import linkStyles from '../link.module.css';
 
 export default function Navigation() {
-  //setting state in here breaks everything no idea why
   return (
-    <nav className={styles.navigation}>
-      <NavLink to="">Home</NavLink>
-      <NavLink to="about">About</NavLink>
-    </nav>
+    <header className={styles.header}>
+      <nav className={styles.navbar}>
+        <a href="/">
+          <img
+            className={styles.navLogoImage}
+            src="/images/Logo.png"
+          />
+        </a>
+        <p className={styles.navLogoText}>Jr. Job Board</p>
+        <ul className={styles.navMenu}>
+          <li className={styles.navItem}>
+            <a href="#">Home</a>
```

Comment:
I like the idea of placeholder hrefs waiting for their assignments.  Smart for getting the layout and base functionality ready to plug in.